src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,20 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post: { title, body, comments, user } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+    </div>
+    <UserInfo user={user} />
```

Comment:
user can be null, so it's better to add the condition
```suggestion
    {user && <UserInfo user={user} />}
```